crates/ssg/src/lib.rs

```diff
@@ -8,8 +8,32 @@ use syntect::html::{ClassStyle, ClassedHTMLGenerator};
 use syntect::parsing::SyntaxSet;
 use syntect::util::LinesWithEndings;
 
+#[derive(Clone, Default, Debug, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[repr(C)]
+pub struct Files {
+    pub path: roc_std::RocStr,
+    pub relpath: roc_std::RocStr,
+    pub url: roc_std::RocStr,
+}
+
+#[derive(Clone, Default, Debug, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[repr(C)]
+pub struct Args {
```

Comment:
The Roc definition for these types is

```roc
Args : {
    inputDir : Path, 
    outputDir: Path,
}
```

Which is lowerCamelCase, not snake_case. Is there automatic case conversion?

Reply:
It was copied from glue. I've change it to snake. Thank you